gradle.properties

```diff
@@ -1,4 +1,6 @@
 # Sets default memory used for gradle commands. Can be overridden by user or command line properties.
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
-org.gradle.daemon=false
\ No newline at end of file
+org.gradle.daemon=false
+
+mcVersion=1.18.2
```

Comment:
I see we are messing with the state of the universe in strange and dangerous ways /s :D

I recently did a commit that made the mod version number a constant in two places because the regex didn't work.. can you fix that too and perhaps add that here as well? I think it would make sense in the scope of this PR.